src/style.css

```diff
@@ -0,0 +1,133 @@
+.page {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 300;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  width: 198px;
+  height: 406px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 40px;
+  display: flex;
+  justify-content: center;
+}
+
+.card__content {
+  margin: 0 16px 5px;
```

Comment:
You don’t need always to add 16px horizontal spaces. You can add paddings for your card